pythonosc/osc_server.py

```diff
@@ -44,18 +49,18 @@ def _is_valid_request(request: List[bytes]) -> bool:
 class OSCUDPServer(socketserver.UDPServer):
     """Superclass for different flavors of OSC UDP servers"""
 
-    def __init__(self, server_address: Tuple[str, int], dispatcher: Dispatcher, bind_and_activate=True) -> None:
+    def __init__(self, server_address: Tuple[str, int], dispatcher: Dispatcher, bind_and_activate: bool = True) -> None:
         """Initialize
 
         Args:
             server_address: IP and port of server
             dispatcher: Dispatcher this server will use
             (optional) bind_and_activate: default=True defines if the server has to start on call of constructor  
         """
-        super().__init__(server_address, _UDPHandler, bind_and_activate)
+        super().__init__(server_address, _UDPHandler, bind_and_activate)  # type: ignore[call-arg]  # https://github.com/python/typeshed/pull/8542
```

Comment:
this PR has been merged now, please fix

Reply:
It has, however the typeshed is distributed within `mypy` so this is blocked until the next `mypy` release.
Once a version of `mypy` with the fix is used we should get a warning that the ignore is no longer valid which will be trivial to fix. Currently this code still passes using the latest `mypy`.

Reply:
okay thanks!